ndk-patches/27c/c++-v1-stdlib.h.patch

```diff
@@ -0,0 +1,27 @@
+--- a/usr/include/c++/v1/stdlib.h	2024-11-08 20:01:18
++++ b/usr/include/c++/v1/stdlib.h	2024-11-08 20:03:38
+@@ -109,6 +109,7 @@
+ #    endif
+ 
+ // MSVCRT already has the correct prototype in <stdlib.h> if __cplusplus is defined
++#    if !defined(__swift__)
```

Comment:
Since this is only defined for Swift, should have no effect on any other project.